index.js

```diff
@@ -12,23 +12,31 @@ start();
 async function start() {
   let min = 1;
   let max = 100;
```

Comment:
You can allow the user to input a max using 
```js
let max = process.argv[2] || 100
```
or you could set it using an await ask(...) and reassigning the value of max based on user input